CONTRIBUTING.md

```diff
@@ -1,75 +1,420 @@
+to `https://agoric.com/documentation/ertp/guide/` displays the VuePress processed `/main/ertp/guide/README.md`.
+While it may seem odd, VuePress expects multiple `README.md` files in a repo; most folders will have one. 
+
+The root README.md file must start with an H1 (`#` in Markdown) header. In fact, all our doc pages should start
+with that. But for READMEs, it's needed to generate search indexes and sidebars.
+
+All directories/folders should have a `README.md` file, even if it's empty. They provide a landing page for
+the folder in the VuePress processed documentation structure. 
+
+Lines with no special treatment are converted into standard HTML paragraph tags.
+
+### Sectioning Pages
+
+VuePress automatically builds search functionality and individual page menus from `h1`, `h2`, and `h3` headers (i.e. Markdown's `#`, `##`, and `###` commands). 
+You must have only **one** `h1` per `.md` file. Be careful not to have too many `h2` and `h3` level headers 
+on one page and that they aren't too long. Otherwise the sidebar menu for the page will be cluttered and hard to read and use.
+
+## Writing Links
+
+VuePress turns Markdown links into HTML links. There are some quirks about how you should write Markdown links.
+
+First, our link checker does **not** check `router-link` style links. Please only use Markdown style links.
+
+Next, your Markdown links should be to the `.md` Markdown files in the Doc repo. VuePress processing changes
+both the `.md` files and links to them to be `.html`.
+
+Use relative links instead of absolute ones for any links to files or folders in the Documentation repo. Relative links
+open in the same browser tab when clicked on, absolute links open a new tab. 
+
+However, there's a trick you can use that's easier than writing a complicated relative link. 
+While it's easy enough to, say, write a relative link to something in
+the same folder as the file you're writing (something like `(./assets/my-diagram.svg)` to include an image), it can be
+difficult to remember/figure out what the right syntax is for relative linking to a file two folders up, in a different upper folder, and then
+two levels down from there on a different branch of the file structure. 
+
+Instead, VuePress considers `main` the top of the file hierarchy. So you can always get to, say, a Glossary entry 
+by just linking to `(/glossary/#allocation)`; its path starting at `main`. Any path starting with just `/` starts
+at `main`. These links also open in the same browser tab. 
+
+VuePress turns every header in a Markdown file to an HTML anchor you can link to, so clicking such a link takes you directly to
+that file location (called *slugifying* by WordPress and other blogging platforms). At the end of the link, 
+append a `#` to the file name, followed by the header text. The header text
+must be altered to be 1) all lower case. 2) All non-alphanumerics, including spaces, are replaced by hyphens (except there aren't any
+trailing hyphens at the end).
```

Comment:
the preface to the list doesn't apply equally to all elements. Please re-word for parallel construction or drop the numbering.